Notepad-Sharp/MainForm.cs

```diff
@@ -142,6 +142,7 @@ void CreateTab(string fileName)
                 tab.mainEditor.OpenFile(fileName);
             }
 
+            tab.mainEditor.KeyDown += new KeyEventHandler(MainForm_KeyDown);
```

Comment:
This was added back to restore the previous hotkey behavior handled by the MainForm (Open File, Insert line prefix, and clear current line).